axiom-eth/src/utils/component/circuit/mod.rs

```diff
@@ -61,7 +61,7 @@ impl CoreBuilderParams for CoreBuilderOutputParams {
 }
 /// Input for CoreBuilder.
 /// TODO: specify its output capacity.
-pub trait CoreBuilderInput<F: Field> = Serialize + DeserializeOwned + Clone + 'static;
```

Comment:
Highlighting this change as it is removing the requirement of Serialize and DeserializeOwned from CoreBuilderInput. This is particularly useful when to we have to input complex types like zkEVM's SuperCircuit.